extensions/producthunt/package.json

```
@@ -0,0 +1,34 @@
+{
+  "name": "producthunt",
+  "title": "Product Hunt",
+  "description": "Get the latest products from the fontpage of Product Hunt.",
+  "icon": "icon.png",
+  "author": "Ragnor Comerford",
+  "commands": [
+    {
+      "name": "frontpage",
+      "title": "Check Today's Product Hunt",
+      "subtitle": "Product Hunt",
```

We tend to minimize repeatetive names here. Both are shown in the root search. How about naming it like this?
```suggestion
      "title": "Check Today's Hunts",
      "subtitle": "Product Hunt",
```